L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc

```diff
@@ -129,12 +129,12 @@ struct L1TStubCompare {
   bool operator()(const trklet::L1TStub& a, const trklet::L1TStub& b) const {
     if (a.x() != b.x())
       return (b.x() > a.x());
-    else {
-      if (a.y() != b.y())
-        return (b.y() > a.y());
-      else
-        return (a.z() > b.z());
-    }
+    else if (a.y() != b.y())
```

Comment:
Are these at low precision? Otherwise I think it may be best to do std::abs() < std::numeric_limits<double>::epsilon here.